sklearn/linear_model/randomized_l1.py

```diff
@@ -484,7 +492,7 @@ class RandomizedLogisticRegression(BaseRandomizedLinearModel):
 
     See also
     --------
-    RandomizedLasso, Lasso, ElasticNet
+    RandomizedLasso, LogisticRegression, ElasticNet
```

Comment:
Same comment as above; you might want to leave `Lasso` mentioned in the documentation.